source/reference/connection-string.txt

```diff
@@ -51,6 +51,8 @@ The components of this string are:
        specific database using these credentials after connecting to
        the :program:`mongod` instance.
 
+       If username or password contains an at-sign ("@") or colon (":") then the username and/or password must be URL encoded. At-sign ("@") is replaced with "%40" (3 characters), while colon (":") is replaced with "%3A" (3 characters).
+
```

Comment:
I opened https://jira.mongodb.org/browse/SPEC-618  for the drivers + https://jira.mongodb.org/browse/NODE-757 since node.js didn't allow escaped colon in the username. I'll think of something for the docs tomorrow.  Thanks much.